force-app/main/default/classes/PS_Request.cls

```diff
@@ -274,9 +274,6 @@ public inherited sharing class PS_Request {
             } else if (endpoint == ElevateEndpoint.COMMITMENT) {
                 value = ENDPOINT_COMMITMENT;
 
-                if (String.isNotBlank(commitmentId)) {
-                    value += '/' + commitmentId;
```

Comment:
This **should** be safe, since COMMITMENT_CANCEL and COMMITMENT_PAUSE are both being set with separate logic.